### PR TITLE
[CBRD-20422] Fixed bad set to compression_length in mr_get_compression_length

### PR DESCRIPTION
**Pull request by lucianlazar2211**
http://jira.cubrid.org/browse/CBRD-20422

In or_put_varchar_internal we have set `compressed_length` to `0` when `compressed_length >= charlen - 8`. But in `mr_get_compression_length` it was set to return when `charlen >= compressed_length + 8` and it should be `charlen > compressed_length + 8`, or `compressed_length < charlen - 8`.

### CODE REVIEW

**Comment by lucianlazar2211** [comment]
Looking into conflicts now.

**Comment by cubridci** [comment]
Can one of the admins verify this patch?

File: src/object/object_primitive.c [hunk 1]
```diff
@@ -16345,7 +16345,7 @@ mr_get_compression_length (const char *string, int charlen)
       goto cleanup;
     }
 
-  if ((lzo_uint) length >= compressed_length + 8)
+  if ((lzo_uint) compressed_length < charlen - 8)
```
Comment (eseokoh) [inline src/object/object_primitive.c]: @lucianlazar2211 

```C
  if (compressed_length < (lzo_uint) (charlen - 8))
```

**Comment by eseokoh** [comment]
@lucianlazar2211 

It looks nice. 
Thank you so much. :)

**Pull request merged**